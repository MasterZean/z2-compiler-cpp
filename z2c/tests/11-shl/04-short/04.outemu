Short{2}
Short{4}
Short{8}
Short{16}
Short{32}
Short{64}
Short{128}
Short{256}
Short{512}
Short{1024}
Short{2048}
Short{4096}
Short{8192}
Short{16384}
Short{-32768}
Short{0}
Short{-2}
Short{-4}
Short{-8}
Short{-16}
Short{-32}
Short{-64}
Short{-128}
Short{-256}
Short{-512}
Short{-1024}
Short{-2048}
Short{-4096}
Short{-8192}
Short{-16384}
Short{-32768}
Short{0}
