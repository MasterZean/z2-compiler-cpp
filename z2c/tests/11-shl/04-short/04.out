(int16)(1) << 1;
(int16)(2) << 1;
(int16)(4) << 1;
(int16)(8) << 1;
(int16)(16) << 1;
(int16)(32) << 1;
(int16)(64) << 1;
(int16)(128) << 1;
(int16)(256) << 1;
(int16)(512) << 1;
(int16)(1024) << 1;
(int16)(2048) << 1;
(int16)(4096) << 1;
(int16)(8192) << 1;
(int16)(16384) << 1;
(int16)(32768) << 1;
(int16)(-1) << 1;
(int16)(-2) << 1;
(int16)(-4) << 1;
(int16)(-8) << 1;
(int16)(-16) << 1;
(int16)(-32) << 1;
(int16)(-64) << 1;
(int16)(-128) << 1;
(int16)(-256) << 1;
(int16)(-512) << 1;
(int16)(-1024) << 1;
(int16)(-2048) << 1;
(int16)(-4096) << 1;
(int16)(-8192) << 1;
(int16)(-16384) << 1;
(int16)(-32768) << 1;
